aikau/src/main/resources/alfresco/header/Title.js

```diff
@@ -129,9 +151,53 @@ define(["dojo/_base/declare",
          }
          if (this.targetUrl)
          {
-            this.textNode.href = AlfConstants.URL_PAGECONTEXT + this.targetUrl;
+            this.makeAnchor(this.targetUrl, this.targetUrlType);
+         }
+         this.alfSubscribe(topics.UPDATE_PAGE_TITLE, lang.hitch(this, this.updatePageTitle));
+      },
+
+      /**
+       * Returns an array containing the selector that identifies the span to wrap in an anchor.
+       * This overrides the [mixed in function]{@link module:alfresco/navigation/_HtmlAnchorMixin}
+       * that just returns an empty array.
+       *
+       * @instance
+       * @since 1.0.32
+       */
+      getAnchorTargetSelectors: function alfresco_header_Title__getAnchorTargetSelectors() {
+         return [".alfresco-header-Title__text"];
+      },
+
+      /**
+       * Handles click events when a [targetUrl]{@link module:alfresco/header/Title#targetUrl} has
+       * been provided.
+       *
+       * @instance
+       * @param {object} evt The click event
+       * @since 1.0.32
+       */
+      onClick: function alfresco_header_Title__onClick(evt) {
+         var targetUrlLocation = this.targetUrlLocation;
+         if (has("mac") && evt.metaKey)
```

Comment:
I don't know - I took the existing code from AlfMenuItemMixin... I didn't add it originally, both should be consistent - there is no compelling reason to change it at the moment (certainly not as part of this issue) but possibly something to consider under a different issue